pom.xml

```diff
@@ -409,15 +409,15 @@
             <id>bamboo924OrLater</id>
             <properties>
                 <bamboo.product.data.path>
-                    ${basedir}/src/test/resources/generated-test-resources-923.zip
+                    ${basedir}/src/test/resources/generated-test-resources-9_2_13.zip
```

Comment:
I'd add a comment such that we're not confused why we have different profiles (just mentioning a change in schema from 9.2.3 to 9.2.4)